### DISCUSSION TITLE
Can we use the command line UI for "q:" also?

### DISCUSSION THREAD

**Discussion by LuizTrMr** [folke/noice.nvim#714] [Q&A]
I wanted to use Noice, but only if I get the same command line UI when pressing ":" and "q:". Is that possible?

**Comment by Sam-programs** [comment]
> if I get the same command line UI when pressing ":" and "q:".

You could get something close to it with the `cmdline` view, but it's not possible to get the original UI without disabling messages too, this is a neovim limitation.
```lua
cmdline = {
 view = "cmdline"
}
```
